README.md

```diff
@@ -1,6 +1,63 @@
 # weBlock
 
-weBlock claims to be an ad-blocker that runs on a server to save client-side processing power, but secretly censors data at the deployer's disposal.
+weBlock claims to be an ad-blocker that runs on a server to save client-side processing power, but could secretly censor data at the deployer's disposal.
 
-This is a collaborative project with [lucasliebe](https://github.com/lucasliebe) and [tiny-fish-T](https://github.com/tiny-fish-T) and is being developed within the scope of a course named *DarkAI* at [HPI](https://hpi.de). The course's goal is to raise awareness about possible adverse effects of AI and provoke more critical thinking in handling software products.
+This is a collaborative project with [lucasliebe](https://github.com/lucasliebe) and [tiny-fish-T](https://github.com/tiny-fish-T) and is being developed within the scope of a course named *DarkAI* at [HPI](https://hpi.de). The course's goal is to raise awareness about possible harms and threats of artificial intelligence and to provoke more critical thinking in handling software products.
+
+## Installation
+
+This project consists of a client and server, which can be installed and run separately. Both actors require a working installation of [Mozilla Firefox](https://www.mozilla.org/en-US/firefox/) and [Zsh](https://www.zsh.org) or [Bash](https://www.gnu.org/software/bash/). In addition, the client requires [Node.js](https://nodejs.org/en/) and [Yarn](https://yarnpkg.com) and the server requires [Python 3](https://www.python.org), [Geckodriver](https://github.com/mozilla/geckodriver/releases) (which is installed by downloading the operating system's respective executable and moving it into the `$PATH`), [Make](https://www.gnu.org/software/make/) and [GCC](https://gcc.gnu.org).
+
+With the above requirements met, the install script can be run in `bash` or `zsh`, e.g.`./install.sh` to install both client as well as server or with an argument `client` or `server` to install only the respective actor.
+
+## Usage
+
+### Client
+
+To run the client, simply go the the `client/` directory and run `yarn start`. This will open an instance of Firefox with the ad-blocking / censoring extension loaded. The toolbar will show weBlock's icon, where the address of your server that has weBlock's server side deployed can be set. By default, this is assumed to be `localhost`.
+
+Browsing with the extension loaded will behave normally, until the circle that will show up on the right side of the URL field is clicked. Clicking it once will put the ad-blocker to work doing it's best to remove advertisements and give you a preview of what content will be censored by coloring it red. Clicking it a second time engage censoring and replace the red text with content the censorer (server) deems as friendly but still contextually relevant.
```

Comment:
Maybe state that only http and https that supply html are supported protocols so it doesnt work for pdf`s for instance